src/server/database/database_response.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any
+
+from . import Message
+
+
+class DatabaseResponse:
+    """Class holding a response of the DatabaseProvider along with the data from cursors.
+
+    If the DatabaseResponse.message is 'OK', the query has been executed successfully.
+    Otherwise, there has been an internal error.
+
+    On 'SELECT' queries, DatabaseResponse.data field is expected to hold some meaningful
+    information. On 'INSERT" queries, usually this field remains empty. It is also expectable to be
+    empty on an error.
+    """
+
+    def __init__(self, message: Message, data: list[Any] = []):
```

Comment:
Są jakieś przypadki, w których to będzie lista czegoś innego, niż krotki?

Reply:
Probably not.

Reply:
Nie powinno się zmienić na list[tuple] ?

Reply:
I think that this unfortunately can't be changed due to typehint mismatches when assigning response.data to variables, i.e. `list[tuple[str, str, str]]`

Reply:
Wydaje Ci się czy jesteś pewien?
U mnie nie podkreśla składni po zmianie, oraz python to toleruje bez problemów

Reply:
As I said, there is a typehint mismatch, raised by mypy specifically

Reply:
Dziwne, u mnie działa xD
A przy okazji: w innych miejscach nie świeci Ci się kod na czerwono? Np w pliku main.py około linii 175? Albo 264?

Reply:
Can you provide the code directly, where you can see the errors? On my end, there is no errors in the current state of the code.

Reply:
Na discrodzie zamieściłem instrukcję odtworzenia blędu